processing/src/test/java/org/apache/druid/segment/virtual/ExpressionSelectorsTest.java

```diff
@@ -231,6 +252,80 @@ public void testCoerceExprToValue()
     );
   }
 
+  @Test
+  public void testIncrementIndexStringSelector() throws IndexSizeExceededException
```

Comment:
Is this the regression test? If so could you add a comment that this is a regression test for whichever the issue number is?

Reply:
added a comment, (there isn't an open issue as far as I could tell)

Reply:
A link to this PR would suffice, IMO.

Reply:
added link to PR